plasmapy/examples/langmuir_analysis.py

```diff
@@ -0,0 +1,89 @@
+# coding: utf-8
+"""
+Analyzing Langmuir
+=========================
+
+Let's analyze various Langmuir probe characteristics using the
+`diagnostics.langmuir` subpackage.
+"""
+
+import astropy.units as u
+from plasmapy.diagnostics.langmuir import Characteristic, swept_probe_analysis
+import pickle
+import numpy as np
+
+######################################################
+"""The first characteristic we analyze is a simple single-probe measurement in
```

Comment:
Take a look at the other `examples` to see the syntax for these examples. The initial docstring is okay, but if you want to write plain text narrative in the middle of the example, prepend each line with a `#` and skip putting these into a multiline string.